en/docs/learn/managing-challenge-questions.md

```diff
@@ -163,3 +163,22 @@ process to the following page (redirection happens only if the user has
 not given answers for the minimum number of forced challenge questions).
     ![answering-challenge-questions.png](../assets/img/using-wso2-identity-server/answering-challenge-questions.png)
 
+### Validating challenge question answers
+
+Configuring a regex pattern for challenge question answers will ensure the uniqueness of an answer and can ensure that more than one question does not have the same answer. To configure a regex pattern for challenge questions answers and validate the answers based on the given regex, follow the instructions below. 
```

Comment:
What does this uniqueness mean, is it among given answers?

Reply:
Yes. Here the uniqueness refers to not having the same answer for two or more questions.